sky/task.py

```diff
@@ -223,10 +224,10 @@ def __init__(
         # https://github.com/python/mypy/issues/3004
         self.num_nodes = num_nodes  # type: ignore
 
-        self.inputs = None
-        self.outputs = None
-        self.estimated_inputs_size_gigabytes = None
-        self.estimated_outputs_size_gigabytes = None
+        self.inputs: Optional[str] = None
+        self.outputs: Optional[str] = None
```

Comment:
Maybe type notation for `set_outputs` and `get_outputs` too? Since it is relevant to this PR

Reply:
Good call! Added. Thanks!